pom.xml

```diff
@@ -113,14 +125,45 @@
 					</instructions>
 				</configuration>
 			</plugin>
+            <!-- Force Maven to run under 1.8, The compiler is set to 1.6, and 1.7 to build the output -->
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-enforcer-plugin</artifactId>
+                <version>3.0.0-M2</version>
+                <executions>
+                    <execution>
+                        <id>enforce-java</id>
+                        <goals>
+                            <goal>enforce</goal>
+                        </goals>
+                        <configuration>
+                            <rules>
+                                <requireJavaVersion>
+                                    <version>1.8.0</version>
+                                </requireJavaVersion>
+                            </rules>
+                        </configuration>
+                    </execution>
+                </executions>
+            </plugin>
 			<plugin>
 				<groupId>org.apache.maven.plugins</groupId>
 				<artifactId>maven-compiler-plugin</artifactId>
-				<version>2.3.2</version>
+				<version>3.8.1</version>
 				<configuration>
 					<source>%%JAVAVERSION%%</source>
 					<target>%%JAVAVERSION%%</target>
+                    <!--<source>1.6</source>
+                    <target>1.6</target>-->

```

Comment:
With this, target is now defined twice

Reply:
ahh, no, sorry, it's a block comment...

Could you remove the commented out code here?